src/capital-project/capital-project.repository.ts

```diff
@@ -103,6 +105,67 @@ export class CapitalProjectRepository {
     }
   }
 
+  async findGeoJsonByManagingCodeCapitalProjectId(
+    params: FindCapitalProjectGeoJsonByManagingCodeCapitalProjectIdPathParams,
+  ): Promise<FindGeoJsonByManagingCodeCapitalProjectIdRepo> {
+    const { managingCode, capitalProjectId } = params;
+    try {
+      return await this.db
+        .select({
+          id: capitalProject.id,
+          managingCode: capitalProject.managingCode,
+          description: capitalProject.description,
+          managingAgency: capitalProject.managingAgency,
+          minDate: capitalProject.minDate,
+          maxDate: capitalProject.maxDate,
+          category: capitalProject.category,
+          sponsoringAgencies: sql<
+            Array<string>
+          >`ARRAY_AGG(DISTINCT ${agencyBudget.sponsor})`,
+          budgetTypes: sql<
+            Array<string>
+          >`ARRAY_AGG(DISTINCT ${agencyBudget.type})`,
+          commitmentsTotal: sum(capitalCommitmentFund.value).mapWith(Number),
+          geometry: sql<string | null>`
+            CASE
+            WHEN 
+              ${capitalProject.liFtMPoly} IS NOT null
+            THEN
+              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPoly}, 4326))	
+            ELSE
+              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPnt}, 4326))
```

Comment:
Pratishta highlighted that we have actually be limiting the geojson to 6 decimal places. (6 decimal places provides enough precision while cutting down the amount of data we're sending)

We achieve this by:

```suggestion
              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPoly}, 4326), 6)	
            ELSE
              ST_asGeoJSON(ST_Transform(${capitalProject.liFtMPnt}, 4326), 6)
```